processor/src/main/kotlin/com/tobrun/datacompat/DataCompatProcessor.kt

```diff
@@ -121,8 +142,10 @@ class DataCompatProcessor(
                 addFunction(
                     FunSpec.builder("toString")
                         .addModifiers(KModifier.OVERRIDE)
+                        // `·` below is a non-breaking space
                         .addStatement(
                             propertyMap.keys.joinToString(
+                                separator = ",·",
```

Comment:
Any chance to add a test with a data class with many fields?

Reply:
I did not want to touch example files too much but perhaps will try indeed 😄

Reply:
added an example with long string field